examples/indexer.py

```diff
@@ -41,11 +46,8 @@
     algod_client, algod_client.send_transaction(ptxn.sign(acct.private_key)), 4
 )
 
-# sleep for a couple seconds to allow indexer to catch up
-import time
-
-time.sleep(2)
-
+# allow indexer to catch up
+indexer_wait_for_round(indexer_client, res["confirmed-round"], 30)
```

Comment:
why 30?

Reply:
In local testing the maximum number of retries I saw was 17. 30 seems like a safe upper bound before bailing out